contracts/Booster.sol

```diff
@@ -230,7 +247,14 @@ contract Booster is ReentrancyGuardUpgradeable {
                 );
             }
 
-            feeToken = _feeToken;
+            feeTokens[_feeToken] = FeeDistro(_distro, lockFees, _executionHash, true);
+
+            allFeeTokens.push(_feeToken);
+
+        } else {
+            feeTokens[_feeToken].distro = _distro;
+            feeTokens[_feeToken].executionHash = _executionHash;
```

Comment:
this could be move to up with these two lines for update the state
 ```
lockFeesIncentive = _lockFeesIncentive;
 stakerLockFeesIncentive = _stakerLockFeesIncentive;

```